packages/curvenote-cli/src/models.ts

```diff
@@ -121,7 +125,9 @@ export class User extends BaseTransfer<string, UserDTO> {
 
   $fromDTO = userFromDTO;
 
-  $createUrl = () => `/users/${this.id}`;
+  $createUrl = () => {
+    return `${this.session.config.editorApiUrl}/users/${this.id}`;
```

Comment:
The existing models are all document/editor centric so we enforce that in their urls